src/changelog.txt

```diff
@@ -1,3 +1,13 @@
+Version: 0.1.14
+Date: 2019-07-06
+  Changes:
+    - changed icon size from 32 to 64
+
+  Added Stacking/Crating for:
+    -none
+    -
```

Comment:
could you exclude the "Added Stacking/Crating for:" paragraph? Since there is nothing added, there is no need for it.

Reply:
okay

Reply:
is done now also considder looking why icons are still not correct

Reply:
Thanks. Im on it but i believe it is due to a in the stacking mod. this needs a bit more time spent debugging.